cmd/runtimetest/main.go

```diff
@@ -459,6 +460,131 @@ func validateMountsExist(spec *rspec.Spec) error {
 	return nil
 }
 
+func isParent(parent, child string) bool {
+	if parent == child {
+		return false
+	}
+
+	parent = filepath.ToSlash(parent)
+	child = filepath.ToSlash(child)
+
+	cparts := strings.Split(child, "/")
+	for i, part := range strings.Split(parent, "/") {
+		if cparts[i] != part {
+			return false
```

Comment:
Sure. I remember that I didn't use it for /some/ reason but I can't remember what it was -- so we'll just switch it.